.travis.yml

```diff
@@ -7,5 +7,7 @@ gemfile:
   - gemfiles/rails_4.1.gemfile
```

Comment:
Guys how is this running in travis? We stopped paying for it.

Reply:
Open source projects are free on travis

Reply:
Ahh cool, just thought it be private by being under mubi